app/controllers/heroku/resources_controller.rb

```diff
@@ -1,12 +1,20 @@
 module Heroku
   class ResourcesController < ApplicationController
+    PLAN_COMMANDS = {
```

Comment:
It looks like this is actually the config that's returned for a plan. Maybe change this to `PLAN_CONFIG`?

Reply:
renamed / moved at 674e37d